modules/ca/src/perl/Makefile

```diff
@@ -53,7 +53,8 @@ endif
 
 Cap5_SRCS = Cap5.xs
 Cap5_LIBS = ca Com
-Cap5_INCLUDES = -I$(shell $(PERL) ../perlConfig.pl archlib)/CORE
+Cap5_INCLUDES = -I$(shell $(PERL) ../perlConfig.pl archlib)/CORE \
+-I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE
```

Comment:
> Till Straumann's nonEpicsThread patch

cf. https://bugs.launchpad.net/epics-base/+bug/1816841

Assuming it is not necessary to correctly run (excluding the non-epics thread case).  Yes, it would be good to keep these two reviews separate.

Reply:
Ok, will do, I needed it because I work on OS-X mohave ;-)

Reply:
I just merged the 3.15 branch into 7.0 so the Mojave issue should be fixed with the HEAD version. You will need to remove your changes to this Makefile to resolve conflicts.